node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric_test.go

```diff
@@ -871,25 +367,25 @@ func TestGetHostsIdByArrayIdentifier(t *testing.T) {
 		ioutilReadFileReturns []ioutilReadFileReturn
 		arrayIdentifier       string
 
-		expErrType        reflect.Type
-		expErr            error
-		expHostList       []int
-		globReturnMatches []string
-		globReturnErr     error
+		expErrType       reflect.Type
+		expErr           error
+		expHostList      []int
+		cmdReturnMatches []string
```

Comment:
why change here?

Reply:
cause the word blob preserve to the path search method

Reply:
please avoid changing this test, it is not related this PR

Reply:
done